lib/unparser/validation.rb

```diff
@@ -25,6 +26,7 @@ def success?
         generated_node
       ].all?(&:right?) && generated_node.from_right.==(original_node.from_right)
```

Comment:
rubocop does a bad job here to think this specific use of the explicit dot is redundant ;)